spec/factories/posts.rb

```diff
@@ -0,0 +1,15 @@
+include ActionDispatch::TestProcess
+FactoryGirl.define do
+  factory :post do
+    title "First post"
+    content "Hello world"
+    image { fixture_file_upload(Rails.root.join('spec', 'photos', 'test.jpg'), 'image/jpg') }
```

Comment:
Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)